skxray/tests/test_core.py

```diff
@@ -349,16 +347,19 @@ def test_radius_to_twotheta():
     dist_sample = 100
     radius = np.linspace(50, 100)
 
-    two_theta = np.array([0.46364761, 0.47177751, 0.47984053, 0.48783644, 0.49576508,
-                          0.5036263, 0.51142, 0.51914611, 0.52680461, 0.53439548,
-                          0.54191875, 0.54937448, 0.55676277, 0.56408372, 0.57133748,
-                          0.57852421, 0.58564412, 0.5926974, 0.59968432, 0.60660511,
-                          0.61346007, 0.62024949, 0.62697369, 0.63363301, 0.6402278,
-                          0.64675843, 0.65322528, 0.65962874, 0.66596924, 0.67224718,
-                          0.67846301, 0.68461716, 0.6907101, 0.69674228, 0.70271418,
-                          0.70862627, 0.71447905, 0.720273, 0.72600863, 0.73168643,
-                          0.73730693, 0.74287063, 0.74837805, 0.75382971, 0.75922613,
-                          0.76456784, 0.76985537, 0.77508925, 0.78027, 0.78539816])
+    two_theta = np.array([0.46364761, 0.47177751, 0.47984053, 0.48783644,
```

Comment:
Change this to use generative tests.  For a given input we _know_ what the answer must be.